.azure-pipelines.yml

```diff
@@ -0,0 +1,89 @@
+#
+# Azure Pipelines configuration for building and testing react on Linux, Windows, and macOS.
+#
+
+trigger:
+- master
+
+jobs:
+- job: LinuxBasic
+  displayName: 'Linux: Basic Tests'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  steps:
+    - template: .azure-pipelines/common/setup.yml
```

Comment:
How do you feel about `.azure-pipelines.yml` living under the `.azure-pipelines` folder? I go back and forth, but am leaning towards putting everything in the folder.

Not that I would base the decision on this, but it would also shorten all the template paths in this YAML file (which there are a lot of) and make this file feel lighter.

Reply:
I could go either way, though I kind of like having the .azure-pipelines.yml at the top level since that's where it is for most other projects.